python/lsst/summit/extras/animation.py

```diff
@@ -26,11 +26,13 @@
 import shutil
 import subprocess
 import uuid
+from typing import Any, List, Tuple
```

Comment:
Don't use `List` or `Tuple`. Use `list` and `tuple`.

Reply:
Nice! Thanks for the tips!